ims.yaml

```diff
@@ -0,0 +1,672 @@
+  contact:
+    name: 'Lisa Mattson (COO), IMS Global'
+    url: 'http://www.imsglobal.org'
+    email: lmattson@imsglobal.org
+  license:
+    name: IMS Global
+    url: 'https://www.imsglobal.org/license.html'
+host: www.imsglobal.org
+basePath: /ims/eduapi/v1p0
+schemes:
+  - http
+  - https
+tags:
+  - name: PersonsManagement
+    description: No description supplied.
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /persons:
+    get:
+      operationId: getAllPersons
+      summary: The REST read request message for the getAllPersons() API call.
+      tags:
+        - PersonsManagement
+      description: ...not supplied...
+      parameters:
+        - name: Limit
+          in: query
+          description: To define the download segmentation value i.e. the maximum number of records to be contained in the response.
+          required: false
+          type: integer
+          format: int32
+          default: 100
+          minimum: 1
+          allowEmptyValue: false
+        - name: Offset
+          in: query
+          description: The number of the first record to be supplied in the segmented response message.
+          required: false
+          type: integer
+          format: int32
+          default: 0
+          minimum: 0
+          allowEmptyValue: false
+        - name: Sort
+          in: query
+          description: 'Identifies the sort criteria to be used for the records in the response. Multiple criteria should be comma-seperated (U+002C COMMA, “,”) and are applied in the order provided. Each field defaults to ascending order, unless prefixed with U+002D HYPHEN-MINUS, “-“ to indicate descending order.'
+          required: false
+          type: array
+          items:
+            type: string
+          collectionFormat: csv
+          allowEmptyValue: false
+        - name: Filter
+          in: query
+          description: 'Identifies filter critera to be used for the records in the response.Multiple criteria should be comma-seperated (U+002C COMMA, “,”).'
+          required: false
+          type: array
+          items:
+            type: string
+          collectionFormat: csv
+          allowEmptyValue: false
+        - name: Fields
+          in: query
+          description: To identify the range of fields that should be supplied in the response message.
+          required: false
+          type: array
+          items:
+            type: string
+          collectionFormat: csv
+          allowEmptyValue: false
+      responses:
+        '200':
+          description: The request was successfully completed and a collection has been returned. This would be accompanied by the 'codeMajor/severity' values of 'success/status' and for a REST binding a HTTP code of '200'.
```

Comment:
Since the status response is gone, could you remove the reference to `codeMajor/severity` and other similar references?